File: spring-best/src/main/java/com/best/springbest/model/Brand.java
package com.best.springbest.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import javax.persistence.*;
@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "brands")
public class Brand {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private String name;

    private String description;

    private String image;

}

File: BEST-App-main/lib/model/brand.dart
class Brand {
  final int id;
  final String name;
  final String description;
  final String image;
  Brand(this.id, this.name, this.description, this.image);

  static Map<String, dynamic> toJson(Brand brand) {
    return {
      'id': brand.id,
      'name': brand.name,
      'description': brand.description,
      'image': brand.image
    };
  }
}
